tests/Fixtures/PathFilters/features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Step\Given;
+
+class FeatureContext implements Context
+{
+    #[Given('/^Some slow step N(\d+)$/')]
+    public function someSlowStepN($num)
```

Comment:
I understand that the name of these steps does not actually matter, but naming them slow, normal and fast may induce users to think that they have something to do with the speed of the test, when it doesn't. I would probable name them something more neutral like `firstStep`, `secondSet` or something like that

Reply:
https://github.com/search?q=repo%3ABehat%2FBehat%20someSlowStepN&type=code

Happy to change it to whatever though...